Сделал шаблонирование основной страницы и первую часть заполения информации клиента.

Kuznetsovka opened the pull request:
Добавил MainController.
Пытался добавить css, но почему так и не подтягивается.

Kuznetsovka committed:
Сделал шаблонирование основной страницы и первую часть заполения информации клиента.
Добавил MainController.
Пытался добавить css, но почему так и не подтягивается.

observant-sun commented on src/main/resources/application.yaml:
```diff
@@ -17,8 +17,11 @@ spring:
+  mvc:
+    servlet:
+      path: /systemair-ac
+    static-path-pattern: /statico/**
```
Ну вот же, ты ж сам написал

Kuznetsovka committed:
Добавил разметку 2-го раздела, но пока не могу решить ошибку.
EL1008E: Property or field 'TypeMontage' cannot be found on object of type 'java.lang.String' - maybe not public or not valid?

Kuznetsovka committed:
Добавил класс TachDataDto и Enum Типы

Kuznetsovka committed:
Решил проблему с css, теперь подтягивается.
Основная страница подгружается. Но проблему в if так и не удалось решить

Kuznetsovka committed:
Подшаманил стили. Объединил фрагменты, т.к. фрагмент действительно не видит Dto в полной мере, взять от туда данные может, но положить нет.
